pkg/apisix/resource_test.go

```diff
@@ -40,7 +40,10 @@ func TestItemUnmarshalJSON(t *testing.T) {
 }
 `
 	err = json.Unmarshal([]byte(emptyData), &items)
-	assert.Equal(t, err.Error(), "unexpected non-empty object")
+	assert.True(t,
+		assert.NotNil(t, err) &&
```

Comment:
No need to combine `assert.True` to use, just add `assert.NotNil(t, err)` alone.

```suggestion
		assert.NotNil(t, err)
		assert.Equal(t, err.Error(), "unexpected non-empty object")
		...
```

The other changes are the same.

Reply:
done